cmd/lambda/leases/create.go

```diff
@@ -159,6 +162,42 @@ func CreateLease(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// start step function
+	var sfnSvc sfniface.SFNAPI
+	err = Services.Config.GetService(&sfnSvc)
+	if err != nil {
+		log.Printf("Failed to retrieve step functions service %s", err)
+	}
+	sfnInput := leasePkg.Lease{
+		AccountID:                &lease.AccountID,
+		PrincipalID:              &lease.PrincipalID,
+		ID:                       &lease.ID,
+		Status:                   leasePkg.Status(string(lease.LeaseStatus)).StatusPtr(),
+		StatusReason:             leasePkg.StatusReason(string(lease.LeaseStatusReason)).StatusReasonPtr(),
+		CreatedOn:                &lease.CreatedOn,
+		LastModifiedOn:           &lease.LastModifiedOn,
+		BudgetAmount:             &lease.BudgetAmount,
+		BudgetCurrency:           &lease.BudgetCurrency,
+		BudgetNotificationEmails: &lease.BudgetNotificationEmails,
+		StatusModifiedOn:         &lease.LeaseStatusModifiedOn,
+		ExpiresOn:                &lease.ExpiresOn,
+		Limit:                    nil,
+		NextAccountID:            nil,
+		NextPrincipalID:          nil,
+	}
+	sfnInputBytes, err := json.Marshal(sfnInput)
+	if err != nil {
+		log.Printf("Failed to retrieve step functions service %s", err)
```

Comment:
Shouldn't we return here? We're getting an error getting sfnInputBytes but then the next line is doing something with them. **I think this needs a change before approving.**

Reply:
@nathanagood fixed this:
https://github.com/Optum/dce/pull/316/commits/e47e92415383e19bb2ec209a2d9ba5652c8061f3